sites/next.skeleton.dev/src/components/docs/FigmaBuyNow.astro

```diff
@@ -0,0 +1,113 @@
+---
+import { Check as IconCheck } from 'lucide-react';
+---
+
+<!-- Load the Polar checkout script -->
+<script src="https://cdn.jsdelivr.net/npm/@polar-sh/checkout@0.1/dist/embed.global.js" defer data-auto-init></script>
+
+<script>
+	const htmlElement = document.documentElement;
+
+	const buyButtons = document.querySelectorAll('[data-polar-checkout]');
+
+	/**
+	 * Updates the theme of the buy buttons based on the presence of the 'dark' class on the HTML element
+	 */
+	function updateTheme() {
+		const theme = htmlElement.classList.contains('dark') ? 'dark' : 'light';
+
+		buyButtons.forEach((button) => {
+			button.setAttribute('data-polar-checkout-theme', theme);
+		});
+	}
+
+	// Initial update of the theme
+	updateTheme();
+
+	// Create a MutationObserver to observe changes to the class attribute of the HTML element
+	new MutationObserver(() => {
+		updateTheme();
+	}).observe(htmlElement, {
+		// Observe changes to attributes
+		attributes: true,
+		// Only observe changes to the class attribute
+		attributeFilter: ['class']
+	});
+</script>
+
+<div class="flex flex-col bg-transparent rounded-container border-[1px] border-surface-200-800 shadow-lg overflow-hidden">
+	<div class="flex flex-col p-6 w-full">
+		<span class="text-xl font-semibold h-full pb-6">Figma UI Kit for Skeleton includes:</span>
```

Comment:
Recommended text:

> Figma designs for all available components

We only refer to things as "v3" for now while we're between the v2/v3 updates.